css/style.css

```diff
@@ -41,40 +33,197 @@ Developer @GA in NYC
   padding: 0 10px 10px 0; /* note the padding around a left floated image */
 }
 
+html {
+  box-sizing: border-box
+}
 /******************************************
 /* BASE STYLES
 /*******************************************/
 
 body {
-     color: #000;
-     font-size: 12px;
+     color: #606161;
+     font-size: 24px;
      line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
+     font-family: 'Open Sans', sans-serif;
+     padding-left: 50px;
+     margin: 0;
+}
+
+header {
+  background: #033048;
+  height: 90px;
+}
+
+h1 {
+  float: left;
+  color: #fff;
+  font-size: 40px;
+  font-weight: 700;
+  margin-left: 75px;
+  margin-right: 100px;
+  padding-top: 20px;
+}
+
+.banner {
+  color: #f9e42e;
+  font-size: 24px;
+  font-weight: 600;
+  padding: 30px;
+  float: right;
+}
+
+h2 {
+  color: #033048;
+  font-size: 40px;
+  font-weight: 600;
+  padding: 30px 0;
+}
+
+#secondtitle {
+  border-top: 1px solid #000000;
+  margin-top: 25px;
+}
+
+.maintext {
+  font-weight: 300;
+}
+
+.readmore {
+  font-weight: 600;
+  margin-left: 475px;
+  padding: 30px;
```

Comment:
To get this text to left align, Instead of using `margin-left` to push it over to the side, I recommend this:

```css
.readmore {
 display: block;
 text-align: right;
 font-weight: 600;
}
```